webui/react/src/components/FilterForm/components/FilterFormStore.ts

```diff
@@ -131,14 +141,17 @@ export class FilterFormStore {
     );
   }
 
-  #updateForm(id: string, patch: PatchFunc<FormField | FormGroup>): void {
+  #updateForm(
+    match: MatchFunc<FormField | FormGroup>,
+    patch: PatchFunc<FormField | FormGroup>,
+    autoSweep?: boolean,
+  ): void {
     this.#formset.update((loadableFilterSet) => {
       return Loadable.map(loadableFilterSet, (filterSet) => {
         // keep updates to a minimum -- ids should be unique, so all following traversal should have no effect
         let hit = false;
```

Comment:
fix: `hit` doesn't seem to be used anymore. if it's not needed it should be removed

Reply:
I think it improves performance a little bit by [returning earlier](https://github.com/determined-ai/determined/blob/33578eced8924345a7a18176f11aadace247f1f0/webui/react/src/components/FilterForm/components/FilterFormStore.ts#L176) if no match

Reply:
Ah I see, I missed that line